requirements/edx/post.txt

```diff
@@ -9,4 +9,9 @@
 # trying to compile lxml as a dependency causes setuptools to go into an
 # infinite loop and run out of memory. Because why would you trust a
 # dependency management tool to manage dependencies for you?
-python-saml==2.1.6
+
```

Comment:
Not necessarily in-scope for this change, but we should remove the direct reference to @mollydb here for the licensing checks -- @nedbat or @jbarciauskas, is there another person or alias we should be using here?